.github/workflows/deploy-prod.yml

```diff
@@ -33,7 +33,8 @@ jobs:
           export_environment_variables: true
       - name: Setup gcloud
         uses: google-github-actions/setup-gcloud@v1
-      - name: Run deploy script
+      - name: Run deploy scripts
         run: |
-          gcloud config set compute/zone us-central1-c
-          gcloud compute ssh openreview@instance-matching-server-gpu --command '/bin/expertise_build.sh' --quiet
+          gcloud config set compute/zone us-central1-b
+          gcloud compute ssh openreview@openreview-expertise-instance --command '/bin/openreview-py_build.sh' --quiet
```

Comment:
Is it OK to always deploy openreview-py when we deploy the expertise?

Reply:
I don't think so, we also need to deploy openreview-py from the tag version not master.

Reply:
I'm not sure, Carlos, should it be here or in the openreview-py repo? https://github.com/openreview/openreview-py/actions I see that is how Celeste deploys openreview-py to the APIs